doc/src/modules/vector/vector_integration.rst

```diff
@@ -0,0 +1,100 @@
+================================
+Applications of Vector Integrals
+================================
+
+To integrate a scalar or vector field over a region, we have to first define a region. SymPy provides three methods for defining a region:
+
+1. Using Parametric Equations.
+
+2. Using Implicit Equation.
+
+3. Using objects of geometry module.
+
+The :func:`~sympy.vector.integrals.vector_integrate` function is used to integrate scalar or vector field over any type of region. It automatically determines the type of integration (line, surface or volume) depending on the nature of the object.
+
+Importing necessary classes and functions:
+
+>>> from sympy import sin, cos, exp, pi
+>>> from sympy.vector import CoordSys3D, ParametricRegion, vector_integrate
+>>> from sympy.abc import r, x, y, z, theta, phi
+>>> C = CoordSys3D('C')
+    
+Calculation of Perimeter, Surface Area and Volume
+=================================================
+
+To calculate the perimeter of circle, we need to define a circle. Let's define it using its parametric equation.
+
+>>> circle = ParametricRegion((4*cos(theta), 4*sin(theta)), (theta, 0, 2*pi))
+
+To calculate the perimeter of the circle, we can integrate it over a scalar field of unit magnitude.
+
+>>> vector_integrate(1, circle)
+8*pi
+
+Suppose a user wants to calculate the perimeter of triangle. Determining the parametric representation of triangle can be difficult. Instead, the user can use an object of :class:`~sympy.geometry.polygon.Polygon` class in the geometry module.
+
+>>> from sympy.geometry import Point, Polygon 
+>>> triangle = Polygon(Point(1, 2), (3, 5), (1,6))
+>>> vector_integrate(1, triangle)
+sqrt(5) + sqrt(13) + 4
+
+To define a solid sphere, we need to use three parameters (r, theta and phi).
+
+>>> sphere = ParametricRegion((r*sin(phi)*cos(theta),r*sin(phi)*sin(theta), r*cos(phi)),
+... (theta, 0, pi), (phi, 0, pi), (r, 0, 3))
```

Comment:
maybe `theta` should range from 0 to `2*pi`?

Reply:
Yes, will fix it.